app/controllers/volunteers_controller.rb

```diff
@@ -66,6 +66,37 @@ def remove
     # for volunteer to remove project, perhaps include a note for why they're
     # no longer available?
   end
+  
+  def add_repairs
+    @repairs = Repair.all
+    # @repairs = Repair.find(params[:id])
+    @volunteer = current_user.volunteer
+  end
+  
+ def add_repair_to_volunteer
+  # @repair = Repair.find(params[:id])
+   volunteer_repair = VolunteerRepair.new
+   volunteer_repair.volunteer_id = current_user.volunteer.id
+   volunteer_repair.repair_id = params[:id]
+  volunteer_repair.status = "open"
+   volunteer_repair.save
+   redirect_to add_repairs_volunteers_path
+ end
+ 
+ def remove_repair
+   @volunteer_repair = VolunteerRepair.find(params[:id])
+    @volunteer_repair.volunteer_id = current_user.volunteer.id
+   @volunteer_repair.destroy
+    redirect_to add_repairs_volunteers_path
+ end
+ 
+ def submit
+    @volunteer_repair = VolunteerRepair.find(params[:id])
+    @volunteer_repair.volunteer_id = current_user.volunteer.id
```

Comment:
You don't need to set the volunteer_repair.volunteer_id again, it is already set.  I don't think I understand what the difference is between a status of open and a status of pending.  My thought was that the volunteer would sign up for a repair and it would immediately be pending, meaning that the administrator could then take some action, moving it to "approved" or something like that.  I don't know why we need a status of "open"